exla/Makefile

```diff
@@ -36,27 +36,29 @@ else
 	CFLAGS += -O3
 endif
 
+NVCC := $(CXX)
+NVCCFLAGS = $(CFLAGS)
 LDFLAGS = -L$(XLA_EXTENSION_LIB) -lxla_extension -shared -fvisibility=hidden
 
 ifeq ($(CROSSCOMPILE),)
 	# Interrogate the system for local compilation
 	UNAME_S = $(shell uname -s)
 
+ifdef ($(EXLA_CPU_ONLY),)
```

Comment:
Should this be ifndef? I see "EXLA_CPU_ONLY is set, skipping nvcc step" in an environment where I haven't set EXLA_CPU_ONLY

Reply:
Yup. Fixing it now.

Reply:
https://github.com/elixir-nx/nx/commit/5ecc82f9cbacdf87712640aa90d685c092918f2e

Fixed it. This is a remnant from the original solution for checking equality. The proper usage for ifdef/ifndef doesn't use the same syntax

Reply:
Whoa! Thanks for the quick fix. Hope you have a good weekend!